react-xmasonry/build.boot

```diff
@@ -19,7 +19,7 @@
 (deftask package []
   (comp
    (download :url (format "https://github.com/ZitRos/react-xmasonry/archive/master.zip")
-             :checksum "E6B128183607A37A561E58E5D913CF4C"
+             :checksum "D9F26537EB811BC06717D08A93E58235"
```

Comment:
Even still, you should also fix the download url to use `+lib-version+` so that the build is reproducible even if library is updated.